svc-bie-kafka/src/main/java/gov/va/vro/services/bie/service/kafka/KafkaConsumer.java

```diff
@@ -0,0 +1,44 @@
+package gov.va.vro.services.bie.service.kafka;
+
+import gov.va.vro.services.bie.config.BieProperties;
+import gov.va.vro.services.bie.service.AmqpMessageSender;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+
+import java.nio.charset.StandardCharsets;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class KafkaConsumer {
+  private final AmqpMessageSender amqpMessageSender;
+  private final BieProperties bieProperties;
+
+  @KafkaListener(
+      topics = {
+        "#{'${kafka.topic.prefix}'}_CONTENTION_BIE_CONTENTION_ASSOCIATED_TO_CLAIM_V02",
```

Comment:
nice this seems much cleaner, should these be in properties (`kafka-topic-to-amqp-exchange-map`) or within the code here?